src/hotspot/share/gc/shared/cardTable.hpp

```diff
@@ -91,6 +91,10 @@ class CardTable: public CHeapObj<mtGC> {
   CardTable(MemRegion whole_heap);
   virtual ~CardTable() = default;
 
+  static bool is_word_aligned(const CardValue* const card) {
```

Comment:
There is a similar method named `is_card_aligned`. So the `is_card_word_aligned` seems more consistent and better.

Reply:
It can be misinterpreted as "is this card word-aligned". In fact, this method doesn't care about "Card". Maybe calling the type "Word"? It's not really related to "Card" -- it just lives inside this class.